src/test/integration/users/User.t.sol

```diff
@@ -72,16 +77,107 @@ contract User is PrintUtils {
+    /// Allocation Manager Methods
+    /// -----------------------------------------------------------------------
+
+    /// @dev Allocates randomly accross the operator set's strategies with a sum of `magnitudeSum`.
+    /// NOTE: Calling more than once will lead to deallocations...
+    function modifyAllocations(OperatorSet memory operatorSet, uint256 magnitudeSum) public virtual createSnapshot {
+        _logM(
+            "modifyAllocations",
+            string.concat(
+                "{avs: ",
+                Logger(operatorSet.avs).NAME_COLORED(),
+                ", operatorSetId: ",
+                cheats.toString(operatorSet.id),
+                ", magnitudeSum: ",
+                _toStringWad(magnitudeSum),
+                "}"
+            )
+        );
+
+        IStrategy[] memory strategies = allocationManager.getStrategiesInOperatorSet(operatorSet);
+
+        AllocateParams[] memory params = AllocateParams({
+            operatorSet: operatorSet,
+            strategies: strategies,
+            newMagnitudes: _randomMagnitudes(magnitudeSum, strategies.length)
+        }).toArray();
+
+        allocationManager.modifyAllocations(params);
+    }
+
+    function allocateAll(
+        OperatorSet memory operatorSet
+    ) public virtual createSnapshot {
+        modifyAllocations(operatorSet, 1.0 ether);
+    }
+
+    function deallocateAll(
+        OperatorSet memory operatorSet
+    ) public virtual createSnapshot {
+        modifyAllocations(operatorSet, 0.0 ether);
+    }
+
+    function registerForOperatorSet(
+        OperatorSet memory operatorSet
+    ) public virtual createSnapshot {
+        _logM(
+            "registerForOperatorSet",
+            string.concat(
+                "{avs: ",
+                Logger(operatorSet.avs).NAME_COLORED(),
+                ", operatorSetId: ",
+                cheats.toString(operatorSet.id),
+                "}"
+            )
+        );
+
+        allocationManager.registerForOperatorSets(
+            RegisterParams({avs: operatorSet.avs, operatorSetIds: operatorSet.id.toArrayU32(), data: ""})
+        );
+    }
+
+    function deregisterFromOperatorSet(
+        OperatorSet memory operatorSet
+    ) public virtual createSnapshot {
+        _logM(
+            "deregisterFromOperatorSet",
+            string.concat(
+                "{avs: ",
+                Logger(operatorSet.avs).NAME_COLORED(),
+                ", operatorSetId: ",
+                cheats.toString(operatorSet.id),
+                "}"
+            )
+        );
+
+        allocationManager.deregisterFromOperatorSets(
+            DeregisterParams({
+                operator: address(this),
+                avs: operatorSet.avs,
+                operatorSetIds: operatorSet.id.toArrayU32()
+            })
+        );
+    }
+
+    function setAllocationDelay(uint32 delay) public virtual createSnapshot {
+        _logM("setAllocationDelay");
+
+        allocationManager.setAllocationDelay(delay);
```

Comment:
We should roll past the allocation configuration delay at the end

Reply:
fixed in [4574f6e](https://github.com/Layr-Labs/eigenlayer-contracts/pull/894/commits/4574f6e615518ffc784794c3be9e53ddbf2f90e6)